### DISCUSSION TITLE
Datetime on Forums are Wrong or not in Local Time

### DISCUSSION THREAD

**Discussion by SyrianBallaS** [RetroAchievements/RAWeb#701] [Ideas]
So I noticed on a forum post I recently did (like an hour ago or so) that I made the post tomorrow, which doesn't even make any sense. I'm going to guess it's in Coordinated Universal Time (UTC) because I just checked it and it says 02:58:32 UTC Friday, March 2, 2018. It's not a big deal just something to point out.

Date and time it shows on forum
![image](https://user-images.githubusercontent.com/18649925/36881237-1bd6372a-1d9b-11e8-973f-b8fa5f52c16d.png)

Date and time shown on my Computer (EST)
![image](https://user-images.githubusercontent.com/18649925/36881345-913712a0-1d9b-11e8-8d98-343446e28527.png)

It's also wrong on forum activity.

The times on the Global Feed are correct however.

**Comment by luchaos** [comment]
This could be solved by allowing users to set a timezone (in e.g. "Europe/Berlin" format to handle DST nicely) - but should be considered a major task on its own as it requires conversion of timestamps all over the site. The Carbon PHP library makes that rather easy.
